inference-engine/tests/unit/cpu/mkldnn_shape_api_test.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "mkldnn_memory.h"
+
+using namespace MKLDNNPlugin;
+using namespace InferenceEngine;
+
+TEST(ShapeAPITest, isStatic) {
+    Shape dynamicShape1(ngraph::PartialShape{{16}, {7, 15}, {-1, -1}, {-1, -1}});
```

Comment:
looks like for all cases you can use the same scheme as in https://github.com/openvinotoolkit/openvino/pull/7431#discussion_r708308944

pass PartialShape and expected result for `isStatic`